src/components/image.html

```diff
@@ -141,11 +141,64 @@
           <div class="component-heading">
             <h3 class="text-large">Image</h3>
           </div>
```

Comment:
I think there is no need of using div as container if it has only 1 element h3. Directly apply the class in div to h3.